bbc/simorgh
# WSTEAMA-1243-p2 Media promos on home pages do not read well without CSS

Resolves JIRA [WSTEAMA-1243](https://jira.dev.bbc.co.uk/browse/WSTEAMA-1243)

Before:
![image](https://github.com/user-attachments/assets/61c33aec-83cf-4c0a-9361-7516f9d0cc57)

After
![image](https://github.com/user-attachments/assets/42392c7f-951d-4480-a8f7-045b187d9016)


Code changes
======

- Used template strings/literals to add a comma and a space between the last word of the title and the word 'Duration' on a media promo


Testing
======

1. Visit the afrique home page
2. Disable all CSS using Web developer chrome extension
3. Locate media promos 
4. There should be a comma followed by a space before the word "duration"

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
